src/saltext/mysql/cache/mysql_cache.py

```diff
@@ -309,17 +309,23 @@ def flush(bank, key=None):
     cur.close()
 
 
-def ls(bank):
+def list_(bank):
     """
-    Return an iterable object containing all entries stored in the specified
-    bank.
+    Return an iterable object containing all entries stored in the specified bank.
     """
+    bank_like = bank + '%'
```

Comment:
note: This should work with and without trailing `/`
```suggestion
    bank = bank.rstrip("/")
    bank_like = bank + "/%"
```

Reply:
I did not know, that in the code they are using a trailing / 
But its easy to handle like this indeed.